### DISCUSSION TITLE
Openmetadata installation with docker unsuccessful with message unhealthy

### DISCUSSION THREAD

**Discussion by Skumar855255** [open-metadata/OpenMetadata#9550] [General]
Hi Team, 

I am new to openmetadata and while doing the installation with docker , it says   "openmetadata-server" is unhealthy. Can anyone please help to find the issue. 

⠿ Network db-data_app_net               Created                                                                                                                          0.0s
 ⠿ Container openmetadata_mysql          Healthy                                                                                                                         16.7s
 ⠿ Container openmetadata_elasticsearch  Started                                                                                                                          0.4s
 ⠿ Container openmetadata_server         Waiting                                                                                                                         40.1s
 ⠿ Container openmetadata_ingestion      Created                                                                                                                          0.0s
container for service "openmetadata-server" is unhealthy

**Comment by pmbrull** [comment]
hi @Skumar855255, apologies for the late response. We might need some more info to help you get started here. I believe it will be faster if you join the [Slack community](https://slack.open-metadata.org/) and give us some details such as which version are you running, how you are starting the containers and any logs from the server.

Thanks
